### DISCUSSION TITLE
Ventura Guidance Revision 1

### DISCUSSION THREAD

**Discussion by robertgendler** [usnistgov/macos_security#188] [General]
Included in this release are updated guidance documents (HTML, PDF, XLS, SCAP) for the NIST SP 800-53r5 Low, Moderate, and High, DISA-STIG, NIST 800-171, CNSSI-1253, CIS macOS Benchmarks Level 1 and 2 Draft, and CIS Critical Security Controls Version 8 baselines for macOS Ventura (13.0).

<hr /><em>This discussion was created from the release <a href='https://github.com/usnistgov/macos_security/releases/tag/ventura_rev1'>Ventura Guidance Revision 1</a>.</em>

**Comment by cspence001** [comment]
Are the updated benchmarks for macOS inclusive of M1, M2, and Intel macs?

**Reply by robertgendler** [reply]
Yes.

There's no difference in settings between M1 and M2. There are some rules that are Apple Silicon(Mx) and Intel specific and marked as such in the tags (i386, arm64)
